### PR TITLE
Make the interval the rtsink rechecks the active flag configurable. 

### PR DESCRIPTION
**Pull request by bowrocker**
Addresses https://github.com/basho/riak_repl/issues/252 by adding app.config configuration:

riak_repl, reactivate_socket_interval, in milliseconds

configuration to schedule_reactivate_socket/1. Leaves default is it was: 10 MS. 

Added test for helper function.

### CODE REVIEW

**Comment by bookshelfdave** [comment]
unless you have any objections, I personally like to see the UOM in a config setting.
would you consider `reactivate_socket_interval_millis` instead of `reactivate_socket_interval`?

File: src/riak_repl2_rtsink_conn.erl [hunk 1]
```diff
@@ -364,13 +367,20 @@ schedule_reactivate_socket(State = #state{transport = T,
             self() ! reactivate_socket,
             State#state{active = false, deactivated = Deactivated + 1};
         false ->
-            %% already deactivated, try again in 10ms
-            erlang:send_after(10, self(), reactivate_socket),
+            %% already deactivated, try again in configured interval, or default
+            ReactivateSockInt = get_reactivate_socket_interval(),
+            lager:debug("reactivate_socket_interval is configured in
```
Comment (bookshelfdave) [inline src/riak_repl2_rtsink_conn.erl]: I believe this will wrap over 2 lines. That doesn't matter much, just pointing it out.

**Comment by bookshelfdave** [comment]
eunit test passes

**Comment by bowrocker** [comment]
pushed changes requested by Dave.

**Comment by bookshelfdave** [comment]
the eunit test can't be run more than once in a Erlang session

**Comment by bookshelfdave** [comment]
Actually, forget the doc changes for now.

**Comment by bookshelfdave** [comment]
:+1:

**Comment by bowrocker** [comment]
Closing..new PR: https://github.com/basho/riak_repl/pull/332

**Pull request closed without merging**